Automattic/wp-calypso
# Update features section in product card i2

### Changes proposed in this Pull Request

This PR updates both style and content of the features section, in the plan cards of the new pricing page.

FIxes 1196341175636977-as-1198920526622494

### Implementation notes

- Slide-outs will be integrated in a separate PR.
- The version checks are not ideal, as mentioned in other platforms. Let's think about how to fix this once i2 is released.

### Testing instructions

- Download the PR
- Concurrently run Calypso and Jetpack cloud
- Make sure the variant `v2 - slide outs` is selected in the `jetpackConversionRateOptimization` A/B test
- Visit the plans page in both Calypso and cloud, expand the plans cards (the first 3 cards), and compare them to the mockups (or screenshots below)
- Switch back to the previous variant and check that the page looks like production

### Screenshots

#### Mockups
<img width="858" alt="Screen Shot 2020-10-26 at 4 11 50 PM" src="https://user-images.githubusercontent.com/1620183/97231005-3d4c6880-17b1-11eb-9a5b-95382a0621cf.png">


#### Implementation
<img width="814" alt="Screen Shot 2020-10-26 at 5 26 39 PM" src="https://user-images.githubusercontent.com/1620183/97230970-302f7980-17b1-11eb-9683-7385ca874e08.png">
